config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
+  get 'home/home'
```

Comment:
Nope! I think it was added by the generator command, but I don't need it because I've changed the root route to be that page anyway.